README.md

```diff
@@ -43,7 +43,30 @@ Both the source and destination connectors share these configuration parameters:
 (*) Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 
 
-Example of a `pipeline.yml` file using `file to activemq classic` and `activemq classic to file` pipelines: 
+### Source configuration
+
+| name | description | required | default value |
+| ---- | ----------- | -------- | ------------- |
+| `clientID`                   | Specifies the JMS clientID which is used in combination with the activemq.subscriptionName to denote a durable subscriber. | No       |               |
+| `dispatchAsync`              | Specifies whether messages should be dispatched synchronously or asynchronously from the producer thread for non-durable topics in the broker. | No       | |
+| `exclusive`                  | Indicates the desire to be the sole consumer from the queue.                                    | No       | |
+| `maximumPendingMessageLimit` | Specifies the upper limit of pending messages allowed for slow consumers on non-durable topics. | No       |               |
+| `noLocal`                    | Indicates if messages sent from the local connection should be excluded from subscriptions.    | No       | |
+| `prefetchSize`               | Determines the maximum number of messages to dispatch to the client before it acknowledges a message. | No       |               |
+| `priority`                   | Specifies the consumer's priority level for weighted dispatching order.                        | No       |               |
+| `retroactive`                | If set to true, makes the subscription retroactive for non-durable topics.                      | No       | |
+| `subscriptionName`           | Specifies the name used for durable topic subscriptions.                                        | No       |               |
+| `selector`                   | Defines a JMS Selector employing SQL 92 syntax as delineated in the JMS 1.1 specification, enabling a filter to be applied on each message associated with the subscription. | No |               |
+
+
+### Destination configuration
+
+There are no specific destination parameters as of writing.
```

Comment:
I'd rather include the complete list of configuration parameters for both the source and destination, even though some of them will be repeated. As a user, if I want to configure the destination I don't care that come options are shared with the source. Also I'd be a bit confused to see that there are no parameters here. We do the same in the kafka connector for instance.